docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -109,6 +109,8 @@ The alert details flyout also lists the number and names of cases to which the a
 
 The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
 
+NOTE: The *Session ID* field provides a unique ID for Linux sessions. To collect it and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+
 The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert with an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert.
```

Comment:
It might be worth noting here that users can click on the alert prevalence count in order to investigate the alerts in timeline.

Reply:
Ah sorry, I did not see that one :D Thanks so much!